migrations/sql/V2021.12.16.8.59__add_project_summary_support_tables.sql

```diff
@@ -0,0 +1,97 @@
+CREATE TABLE IF NOT EXISTS project_summary_contact (
+    project_summary_contact_guid        uuid DEFAULT gen_random_uuid()       PRIMARY KEY,
+    project_summary_guid                uuid                                    NOT NULL,
+    "name"                              character varying(200)                  NOT NULL,
+    job_title                           character varying(100)                          ,
```

Comment:
@BenHarrington why is the name, and the description (below) in quotations?

Reply:
Name and description are keywords in SQL, this is just an extra safety for the DDL as sometimes namespace conflicts can occur when referencing them, having said that you should be able to reference them with or without the quotes in almost all cases.